luigi/contrib/kubernetes.py

```diff
@@ -213,10 +213,15 @@ def poll_interval(self):
         """How often to poll Kubernetes for job status, in seconds."""
         return self.__DEFAULT_POLL_INTERVAL
 
+    @property
+    def pod_creation_wait_interal(self):
+        """Delay for initial pod creation for just submitted job"""
+        return 5
+
     def __track_job(self):
         """Poll job status while active"""
         while not self.__verify_job_has_started():
-            time.sleep(self.__DEFAULT_POLL_INTERVAL)
+            time.sleep(self.poll_interval)
```

Comment:
👍

Reply:
I believe you have a formatting bug on L287 of this patch. You want to use '%s' instead of '{}' and '%' instead of comma to be stylistically consistent with the rest of the file.

I too am running into this on one of our more established pipelines btw.

Reply:
Thanks, i fixed it

Reply:
Please note that with your patch, things run better but now I am getting an issue with dependencies (Unfulfilled exceptions) when I raise workers > 1. I have no idea why.